addons/l10n_id_efaktur/models/efaktur.py

```diff
@@ -77,10 +83,29 @@ def _constrains_min_max(self):
             ], limit=1):
                 raise ValidationError(_('Efaktur interleaving range detected'))
 
-    @api.depends('min', 'max')
-    def _compute_available(self):
-        for record in self:
-            record.available = 1 + int(record.max) - int(record.min)
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            if not vals.get('available'):
+                vals['available'] = 1 + int(vals['max']) - int(vals['min'])
```

Comment:
you're right actually, supposedly the range (and hence min+max) should be provided by the government.

But why not require the min and max in calculation for available?

Reply:
I meant, the two fields are not set as required so they could not be in the vals (or I guess it would be 0/computed/...)
Maybe making these required would make sense

Reply:
ah I see, yes will make them required and remove the compute